lib/pharos/kube/client.rb

```diff
@@ -71,6 +71,17 @@ def get_resource(resource)
 
         get_entity(definition.resource_name, resource.metadata.name, resource.metadata.namespace)
       end
+
+      # @param resource [Kubeclient::Resource]
+      def delete_resource(resource, propagationPolicy: 'Foreground')
+        definition = entity_for_resource(resource)
+
+        delete_entity(definition.resource_name, resource.metadata.name, resource.metadata.namespace, delete_options: {
+          apiVersion: 'v1',
```

Comment:
Does the hardcoded `v1` work here?

Reply:
This is from the original [`Pharos::Kube::Resource#delete`](https://github.com/kontena/pharos-cluster/pull/326/files#diff-2582468926bd28a4186bc42425aa7b42L68)... AFAIK it's the version of the `DeleteOptions` itself, not of the resource.

Reply:
Right.